docs/flow/modin/data_management/functions.rst

```diff
@@ -17,25 +17,25 @@ Function module provides templates for this type of function:
 * Reduction functions — Function that reduces specified axis into a scalar, but requires knowledge about the whole axis. Be aware that providing this knowledge may be expensive because the execution engine has to concatenate partitions along the specified axis. Also, note that the execution engine expects that the reduction function returns a scalar.
 * Default-to-pandas functions — Do fallback to pandas for passed function.
 
-Each template represented with a class with the corresponding name and implements ``register`` method, which takes functions to apply in an appropriate way and instantiate the related template. Passed functions will be evaluated under deserialized partitions (some pandas object: `DataFrame`, `Series`, `GroupByObject` ...).
+Each template represented with a class with the corresponding name and implements ``register`` method, which takes functions to apply in an appropriate way and instantiate the related template. Functions that are passed to the ``register`` will be executed under deserialized partitions which are represented by one of the pandas object: ``pandas.DataFrame``, ``pandas.Series`` or ``pandas.DataFrameGroupbyObject`` (depends on the template type).
```

Comment:
Hmm, I thought a partition is always a pandas DataFrame... but that it was also an implementation detail. Is it not so? And why should we care which object is used as partition implementation?

Reply:
It seems that I wasn't precise again in that formulation. Yes, a partition is always a `pandas.DataFrame`, but the template function may modify deserialized frame (`pandas.DataFrame`) before passing it to the registered function (for example, build groupby object from it or squeeze).

I've tried to rephrase it again)